web/src/main/webapp/xslt/common/index-utils.xsl

```diff
@@ -153,6 +153,17 @@
       "title": "<xsl:value-of select="util:escapeForJson($recordTitle)"/>",
       "origin": "<xsl:value-of select="normalize-space($origin)"/>"
       }</recordLink>
+    <xsl:variable name="fieldName"
+                  select="concat('recordLink_', $type, string-join($otherProperties//p[@name != '']/concat(@name, @value), '_'))"/>
+    <xsl:element name="{$fieldName}">
+      <xsl:value-of select="util:escapeForJson($recordTitle)"/>
+    </xsl:element>
+    <xsl:element name="{$fieldName}_uuid">
+      <xsl:value-of select="normalize-space($uuid)"/>
+    </xsl:element>
+    <xsl:element name="{$fieldName}_url">
+      <xsl:value-of select="normalize-space($url)"/>
+    </xsl:element>
```

Comment:
Should not be checked if `$otherProperties` is provided?

Reply:
If `$otherProperties` is not provided, the field name is only `recordLink_$type`